cont3xt/vueapp/src/components/integrations/IntegrationTidbit.vue

```diff
@@ -6,12 +6,12 @@
 
     </b-tooltip><!--  /tooltip  -->
 
-    <label v-if="labeled" :for="id" class="text-warning">
```

Comment:
Long values overflow the itype container, but not the panel. It would be nice if they got cut off by their immediate parent instead of the panel.